src/env.h

```diff
@@ -116,197 +116,200 @@ struct PackageConfig {
-  V(signal_string, "signal")                                                  \
-  V(sink_string, "sink")                                                      \
-  V(size_string, "size")                                                      \
-  V(sni_context_err_string, "Invalid SNI context")                            \
-  V(sni_context_string, "sni_context")                                        \
-  V(source_string, "source")                                                  \
-  V(stack_string, "stack")                                                    \
-  V(start_time_string, "startTime")                                           \
-  V(status_string, "status")                                                  \
-  V(stdio_string, "stdio")                                                    \
-  V(subject_string, "subject")                                                \
-  V(subjectaltname_string, "subjectaltname")                                  \
-  V(syscall_string, "syscall")                                                \
-  V(thread_id_string, "threadId")                                             \
-  V(ticketkeycallback_string, "onticketkeycallback")                          \
-  V(timeout_string, "timeout")                                                \
-  V(tls_ticket_string, "tlsTicket")                                           \
-  V(ttl_string, "ttl")                                                        \
-  V(type_string, "type")                                                      \
-  V(uid_string, "uid")                                                        \
-  V(unknown_string, "<unknown>")                                              \
-  V(url_string, "url")                                                        \
-  V(username_string, "username")                                              \
-  V(valid_from_string, "valid_from")                                          \
-  V(valid_to_string, "valid_to")                                              \
-  V(value_string, "value")                                                    \
-  V(verify_error_string, "verifyError")                                       \
-  V(version_string, "version")                                                \
-  V(weight_string, "weight")                                                  \
-  V(windows_hide_string, "windowsHide")                                       \
-  V(windows_verbatim_arguments_string, "windowsVerbatimArguments")            \
-  V(wrap_string, "wrap")                                                      \
-  V(writable_string, "writable")                                              \
-  V(write_host_object_string, "_writeHostObject")                             \
-  V(write_queue_size_string, "writeQueueSize")                                \
-  V(x_forwarded_string, "x-forwarded-for")                                    \
+#define PER_ISOLATE_STRING_PROPERTIES(V)                                       \
```

Comment:
Can we make the spacing the same as before on all of these lines so that the diff will be much easier to view?

Reply:
make `format-cpp` just kinda... did that. i'm not sure how to undo it.

Reply:
They would have to be done manually. +1 on reverting this particular change.